src/commands/dev.ts

```diff
@@ -384,6 +391,10 @@ export default class Dev extends BaseCommand {
       dependency_manager.environment = 'local-production';
     }
 
+    if (flags.debug === 'false') {
+      dependency_manager.environment = 'local-production';
```

Comment:
could you add a boolean `debug: true` on the local dependency_manager?

```ts
if (flags.debug === 'false') {
  dependency_manager.debug = false;
}
```

I don't want debug to be tied to environments anymore. Then in the manager you can use `this.debug` to determine whether to merge.